core/block.ts

```diff
@@ -1309,7 +1309,7 @@ export class Block implements IASTNodeLocation, IDeletable {
    * @return Text of block.
    */
   toString(opt_maxLength?: number, opt_emptyToken?: string): string {
-    let text = [];
+    const text = [];
```

Comment:
I think I'd have a mall preference to rename this variable, and keep `text` as the name of the combined `string` variable.  But (unlike the similar case in `generator.ts` below) `lines` isn't an accurate name, so maybe you can suggest something?  But if no better idea surfaces then then the choice you've made here is fine.

Reply:
Yes, `tokens` sounds good.